src/resources/views/crud/fields/upload_multiple.blade.php

```diff
@@ -2,6 +2,17 @@
     $field['wrapper'] = $field['wrapper'] ?? $field['wrapperAttributes'] ?? [];
     $field['wrapper']['data-init-function'] = $field['wrapper']['data-init-function'] ?? 'bpFieldInitUploadMultipleElement';
     $field['wrapper']['data-field-name'] = $field['wrapper']['data-field-name'] ?? $field['name'];
+
+	if(isset($field['parentFieldName'])) {
+		if(!empty(old())) {
+			$field['value'] = array_merge(
+								explode(',',Arr::get(old(), '_order_'.square_brackets_to_dots($field['name'])) ?? ''),
+								Arr::get(old(), 'clear_'.square_brackets_to_dots($field['name'])) ?? [],
```

Comment:
I wish we would prefix all accessory variables with `_`. To know that EVERYTHING that starts with `_` is not an actual field, but something that the system uses for some purpose.

That would simplify our request-stripping logic, we can just strip out everything that starts with `_` by default.

It's a breaking change, but a minor one - unlikely to break anyone's app, unless they've published any of the fields that use this OR have fields in the BD that start with underscore. Then again, it is a breaking change, so it's one more item in the upgrade guide 🤷‍♂️

🟡SHOULD/🔵COULD